### PR TITLE
Adapt AzureRM6.13 tools to make AzureRM sign pipeline workable

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description

- upload ExpandModules.ps1 and PackModules.ps1 from Az master branch
- bug fix for tools/PackModules.ps1
- Skip Stack publish

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: tools/ExpandModules.ps1 [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+#This script will expand Az.*.nupkg under "artifacts" to a temporary folder "artifacts/tmp"
+
+param(
+    [Parameter(Mandatory = $false, Position = 1)]
```
Comment (erich-wang) [inline tools/ExpandModules.ps1:18]: Position should be 0?

File: tools/PackModules.ps1 [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+#This script will pack build artifacts under temporary folder "artifacts/tmp" and output Az.*.nupkg to "artifacts"
+
+param(
+    [Parameter(Mandatory = $false, Position = 1)]
+    [string]$Artifacts,
+
+    [Parameter(Mandatory = $false, Position = 5)]
+    [string]$NugetExe
+)
```
Comment (erich-wang) [inline tools/PackModules.ps1:23]: Position?

File: tools/PackModules.ps1 [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+.SYNOPSIS Write out to a file using UTF-8 without BOM.
+.PARAMETER File
+The file to write the contents too.
+.PARAMETER Text
+The new file contents.
+#>
+function Out-FileNoBom {
+    param(
+        [System.string]$File,
+        [System.string]$Text
```
Comment (erich-wang) [inline tools/PackModules.ps1:41]: nit: suggest to use string for consistent style

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: tools/ExpandModules.ps1 (hunk 1 above)
Comment (BethanyZhou) [reply tools/ExpandModules.ps1:18]: Position=1 is fine

Comment (BethanyZhou) [reply tools/ExpandModules.ps1:18]: Resolved

File: tools/PackModules.ps1 (hunk 3 above)
Comment (BethanyZhou) [reply tools/PackModules.ps1]: Resolved

**Review by erich-wang — APPROVED** [review]

**Pull request merged**